test/fixtures/wallet-web/docker-compose-server.yml

```diff
@@ -93,7 +93,6 @@ services:
       - AUTH_REST_COOKIE_ENC_KEY=/etc/keys/session_cookies/enc.key
       - AUTH_REST_STATIC_IMAGES=/etc/static/images
       - GNAP_ACCESS_POLICY=/etc/static/config/access_policy.json
-      - GNAP_DEV_MODE=true
```

Comment:
Removed as part of https://github.com/trustbloc/wallet/pull/1823/files#diff-919db23f2b4049ad12b17c6291adfaf2e2d76cfd25f6f85a534e67cd1c3ef388L96